resources/dicts/thoughts/dead/darkforest/kitten.json

```diff
@@ -9,15 +9,27 @@
             "Feels hopeless, not understanding what {PRONOUN/m_c/subject} did wrong",
             "Closes {PRONOUN/m_c/poss} eyes, hoping to see the stars once they're opened",
             "Is terrified of the shadows surrounding {PRONOUN/m_c/object}",
-            "Feels as if {PRONOUN/m_c/subject} {VERB/m_c/have/has} been abandoned"
+            "Feels as if {PRONOUN/m_c/subject} {VERB/m_c/have/has} been abandoned",
+            "Misses the warmth of the nursery",
+            "Feels like {PRONOUN/m_c/possessive} innocent is gone",
```

Comment:
innocence rather than innocent! also, every instance of "possessive" only needs to be "poss" : )